dvc/path_info.py

```diff
@@ -3,6 +3,8 @@
 import sys
 import os
 import posixpath
+from collections import Sequence
```

Comment:
This will stop working in Python 3.8 and is deprecated for some time. However, `from collections.abc import Sequence`, which is the upgrade path, won't work in Python 2.7

Do you really need to subclass from it?

Reply:
I think I brought it from the pathlib itself. Not that we really need it. Haven't seen anyone doing type checks with Sequence tbh.